ipatests/test_integration/test_installation.py

```diff
@@ -182,10 +182,21 @@ class TestInstallWithCA1(InstallTestBase1):
     def remove_named(cls, host):
         # remove the bind package and make sure the named user does not exist.
         # https://pagure.io/freeipa/issue/8936
-        result = host.run_command(['id', 'named'], raiseonerr=False)
+        result = host.run_command(
+            ['getent', 'passwd', constants.NAMED_USER],
+            raiseonerr=False
+        )
         if result.returncode == 0:
-            tasks.uninstall_packages(host, 'bind')
+            tasks.uninstall_packages(host, ['bind'])
             host.run_command(['userdel', constants.NAMED_USER])
+
+        result = host.run_command(
+            ['getent', 'group', constants.NAMED_GROUP],
+            raiseonerr=False
+        )
+        if result.returncode == 0:
+            host.run_command(['groupdel', constants.NAMED_GROUP])
```

Comment:
This is normally not necessary.
Since USERGROUPS_ENAB is set to yes in login.defs, userdel removes the eponymous group.

Reply:
Ah, perfect! I removed my commit.